src/pages/en/guides/upgrade-to/v2.mdx

```diff
@@ -0,0 +1,405 @@
+
+#### What should I do?
+
+Rename an existing `src/content/` folder to avoid conflicts. This folder, if it exists, can now only be used for [content collections](/en/guides/content-collections/).
+
+### Changed: `Astro.site` trailing slash
+
+In 1.x, Astro ensured the URL you set as `site` in `astro.config.mjs` always had a trailing slash when accessed using `Astro.site`.
+
+Astro 2.0 no longer modifies the value of `site`. `Astro.site` will use the exact value defined, and a trailing slash must be specified if desired.
+
+#### What should I do?
+
+ In `astro.config.mjs`, add a trailing slash to the URL set in `site`. 
+
+```js del={5} ins={6}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  site: 'https://example.com',
+  site: 'https://example.com/',
+});
+```
+
+### Changed: `_astro/` folder for build assets
+
+In 1.x, assets were built to various locations, including `assets/`, `chunks/` and the root of the build output.
+
+Astro 2.0 moves and unifies the location of all build output assets to a new `_astro/` folder. 
+
+<FileTree>
+- dist/
+  - _astro
+    - client.9218e799.js
+    - index.df3f880e0.css
+</FileTree>
+
+You can control this location with the [new `build.assets` configuration option](/en/reference/configuration-reference/#buildassets).
+
+#### What should I do?
+
+Update your deployment platform configuration if it relies on the location of these assets.
+
+### Changed: Markdown plugin configuration
+
+#### Removed: `extendDefaultPlugins`
+
+In 1.x, Astro used `markdown.extendDefaultPlugins` to re-enable Astro's default plugins when adding your own Markdown plugins.
+
+Astro 2.0 removes this configuration option entirely because its behavior is now the default.
+
+Applying remark and rehype plugins in your Markdown configuration  **no longer disables Astro's default plugins**. GitHub-Flavored Markdown and Smartypants are now applied whether or not custom `remarkPlugins` or `rehypePlugins` are configured.
+
+
+##### What should I do?
+
+Remove `extendDefaultPlugins` in your configuration. This is now Astro's default behavior in 2.0, and you can delete this line without any replacement.
+
+```js del={6}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  markdown: {
+    extendDefaultPlugins,
+  }
+});
+```
+
+#### Added: `gfm` and `smartypants`
+
+In 1.x, you could choose to disable both of Astro's default Markdown plugins (GitHub-Flavored Markdown and SmartyPants) by setting `markdown.extendDefaultPlugins: false`.
+
+Astro 2.0 replaces `markdown.extendDefaultPlugins: false` with separate Boolean options to control each of Astro's built-in default Markdown plugins individually. These are enabled by default, and can be set to `false` independently.
```

Comment:
I generally prefer to put the adverb mid-sentence instead of at the end, as it makes it a bit easier to read IMO. Also, small comma change.

```suggestion
Astro 2.0 replaces `markdown.extendDefaultPlugins: false` with separate Boolean options to individually control each of Astro's built-in default Markdown plugins. These are enabled by default and can be set to `false` independently.
```